Changes to support adding users

Create dependency for config state on configd.service

---

--- a/systemd/cloud-config.service.tmpl
+++ b/systemd/cloud-config.service.tmpl
@@ -1,14 +1,14 @@
 ## template:jinja
 [Unit]
 Description=Apply the settings specified in cloud-config
-After=network-online.target cloud-config.target
+After=network-online.target cloud-config.target config-loaded.target
 After=snapd.seeded.service
 Wants=network-online.target cloud-config.target
 ConditionKernelCommandLine=cloud-init
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/cloud-init modules --mode=config
+ExecStart=/opt/vyatta/sbin/lu -user configd -- /usr/bin/cloud-init modules --mode=config
 RemainAfterExit=yes
 TimeoutSec=0
 
--- a/systemd/cloud-final.service.tmpl
+++ b/systemd/cloud-final.service.tmpl
@@ -12,7 +12,7 @@
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/cloud-init modules --mode=final
+ExecStart=/opt/vyatta/sbin/lu -user configd -- /usr/bin/cloud-init modules --mode=final
 RemainAfterExit=yes
 TimeoutSec=0
 KillMode=process
